src/test/ui/traits/pointee-tail-is-generic.rs

```diff
@@ -0,0 +1,12 @@
+// check-pass
+
+#![feature(ptr_metadata)]
+
+fn a<T>() {
```

Comment:
Can you also add a test to make sure `fn a<T: ?Sized> { b::<T>(); }` continues to generate an error?